File: pinyougou-interface/src/main/java/cn/itcast/core/service/OrderService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.order.Order;
import entity.PageResult;

import java.util.List;
import java.util.Map;

public interface OrderService {
    void add(Order order);


    List<Map> findAll(String name);

    PageResult search(Integer page, Integer rows, Order order);

    void updateStatus(Long[] ids);
}

File: pinyougou-web-shop/src/main/webapp/js/service/orderService.js
//服务层
app.service('orderService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../order/findAll.do');
	}
    this.search = function(page,rows,searchEntity){
        return $http.post("../order/search.do?pageNo="+page+"&pageSize="+rows,searchEntity);
    }



    this.updateStatus = function(ids){
        return $http.get('../order/updateStatus.do?ids='+ids);
    }
});
